taxonomy.php

```diff
@@ -16,7 +16,7 @@
 $taxonomy = get_queried_object();
 
 $context['taxonomy'] = $taxonomy;
-$context['wp_title'] = $taxonomy->name;
-$context['og_description'] = $taxonomy->description;
+$context['wp_title'] = $taxonomy ? $taxonomy->name : '';
```

Comment:
Should we even do anything if the `taxonomy` is undefined? 🤔

Reply:
It's weird that this template would be called then

Reply:
I know and agree with you.
However, as we cannot see the traces of these errors on Sentry ([here](https://sentry.greenpeace.org/organizations/greenpeace-org/issues/202/?project=2) and [here](https://sentry.greenpeace.org/organizations/greenpeace-org/issues/452/?project=2)) I just focused on fixing the reported warning and avoiding additional changes with unexpected consequences.
What do you think?

Reply:
Yeah I think that makes sense!